### PR TITLE
Describe onSelectionChanging event

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: api-reference/10 UI Components/CollectionWidget/1 Configuration/onSelectionChanged.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ default: null
 ---
 ---
 ##### shortDescription
-A function that is executed when a collection item is selected or selection is canceled.
+A function that is executed after a collection item is selected or selection is canceled.
```
Comment (Abadzhev) [inline api-reference/10 UI Components/CollectionWidget/1 Configuration/onSelectionChanged.md]: I think "called" works better with "function" than "executed". If we can use "called", then let's do it. 

 ```suggestion
A function that is called after a collection item is selected or selection is canceled.
```

File: api-reference/10 UI Components/CollectionWidget/1 Configuration/onSelectionChanging.md [hunk 2]
```diff
@@ -5,28 +5,28 @@ default: null
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+A function that is executed before a collection item is selected or selection is canceled.
```
Comment (Abadzhev) [inline api-reference/10 UI Components/CollectionWidget/1 Configuration/onSelectionChanging.md]:
```suggestion
A function that is called before a collection item is selected or selection is canceled.
```

File: api-reference/10 UI Components/CollectionWidget/1 Configuration/onSelectionChanged.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ default: null
 ---
 ---
 ##### shortDescription
-A function that is executed when a collection item is selected or selection is canceled.
+A function that is executed after selection changes.
```
Comment (Abadzhev) [inline api-reference/10 UI Components/CollectionWidget/1 Configuration/onSelectionChanged.md]: I would use the verb "call". Usually you would see a phrase "call a function", but I don't think "execute a function" is that common. 

```suggestion
A function that is called after selection changes.
```

File: api-reference/10 UI Components/CollectionWidget/1 Configuration/onSelectionChanging.md [hunk 4]
```diff
@@ -5,28 +5,28 @@ default: null
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+A function that is executed before selection changes.
```
Comment (Abadzhev) [inline api-reference/10 UI Components/CollectionWidget/1 Configuration/onSelectionChanging.md]:
```suggestion
A function that is called before selection changes.
```

File: api-reference/10 UI Components/CollectionWidget/1 Configuration/onSelectionChanging.md [hunk 5]
```diff
@@ -5,28 +5,28 @@ default: null
...
-<!-- Description goes here -->
+The UI component's instance.
 
 ##### field(e.element): DxElement
-<!-- Description goes here -->
+#include common-ref-elementparam with { element: "UI component" }
 
 ##### field(e.removedItems): Array<any>
-<!-- Description goes here -->
+The data of the items whose selection has been canceled.
```
Comment (Abadzhev) [inline api-reference/10 UI Components/CollectionWidget/1 Configuration/onSelectionChanging.md]: "items whose selection has been canceled" - this phrase is a little complicated. You wouldn't say "I canceled this item's selection". You would say "I deselected this item" or "I removed this item from selection". 

```suggestion
The data of the items removed from selection.
```

or 

```suggestion
The data of the items that have been deselected.
```

P.S. I use the first variant in all other edits. But if you like the second better, you can use it instead.

File: api-reference/10 UI Components/dxList/1 Configuration/onSelectionChanged.md [hunk 6]
```diff
@@ -4,7 +4,7 @@ type: function(e)
 ---
 ---
 ##### shortDescription
-A function that is executed when a collection item is selected or selection is canceled.
+A function that is executed after selection changes.
```
Comment (Abadzhev) [inline api-reference/10 UI Components/dxList/1 Configuration/onSelectionChanged.md]:
```suggestion
A function that is called after selection changes.
```

File: api-reference/10 UI Components/dxList/1 Configuration/onSelectionChanging.md [hunk 7]
```diff
@@ -4,25 +4,25 @@ type: function(e)
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+A function that is executed before selection changes.
```
Comment (Abadzhev) [inline api-reference/10 UI Components/dxList/1 Configuration/onSelectionChanging.md]:
```suggestion
A function that is called before selection changes.
```

File: api-reference/10 UI Components/dxList/1 Configuration/onSelectionChanging.md [hunk 8]
```diff
@@ -4,25 +4,25 @@ type: function(e)
...
-<!-- Description goes here -->
+The UI component's instance.
 
 ##### field(e.element): DxElement
-<!-- Description goes here -->
+#include common-ref-elementparam with { element: "UI component" }
 
 ##### field(e.removedItems): Array<any>
-<!-- Description goes here -->
+The data of the items whose selection has been canceled.
```
Comment (Abadzhev) [inline api-reference/10 UI Components/dxList/1 Configuration/onSelectionChanging.md]:
```suggestion
The data of the items removed from selection.
```

File: api-reference/10 UI Components/dxTabPanel/1 Configuration/onSelectionChanged.md [hunk 9]
```diff
@@ -4,7 +4,7 @@ type: function(e)
 ---
 ---
 ##### shortDescription
-A function that is executed when a collection item is selected or selection is canceled.
+A function that is executed after selection changes.
```
Comment (Abadzhev) [inline api-reference/10 UI Components/dxTabPanel/1 Configuration/onSelectionChanged.md]:
```suggestion
A function that is called after selection changes.
```

File: api-reference/10 UI Components/dxTabPanel/1 Configuration/onSelectionChanging.md [hunk 10]
```diff
@@ -4,25 +4,25 @@ type: function(e)
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+A function that is executed before selection changes.
```
Comment (Abadzhev) [inline api-reference/10 UI Components/dxTabPanel/1 Configuration/onSelectionChanging.md]:
```suggestion
A function that is called before selection changes.
```

File: api-reference/10 UI Components/dxTabPanel/1 Configuration/onSelectionChanging.md [hunk 11]
```diff
@@ -4,25 +4,25 @@ type: function(e)
...
-<!-- Description goes here -->
+The UI component's instance.
 
 ##### field(e.element): DxElement
-<!-- Description goes here -->
+#include common-ref-elementparam with { element: "UI component" }
 
 ##### field(e.removedItems): Array<any>
-<!-- Description goes here -->
+The data of the items whose selection has been canceled.
```
Comment (Abadzhev) [inline api-reference/10 UI Components/dxTabPanel/1 Configuration/onSelectionChanging.md]:
```suggestion
The data of the items removed from selection.
```

File: api-reference/10 UI Components/dxTabs/1 Configuration/onSelectionChanged.md [hunk 12]
```diff
@@ -4,7 +4,7 @@ type: function(e)
 ---
 ---
 ##### shortDescription
-A function that is executed when a collection item is selected or selection is canceled.
+A function that is executed after selection changes.
```
Comment (Abadzhev) [inline api-reference/10 UI Components/dxTabs/1 Configuration/onSelectionChanged.md]:
```suggestion
A function that is called after selection changes.
```

File: api-reference/10 UI Components/dxTabs/1 Configuration/onSelectionChanging.md [hunk 13]
```diff
@@ -4,25 +4,25 @@ type: function(e)
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+A function that is executed before selection changes.
```
Comment (Abadzhev) [inline api-reference/10 UI Components/dxTabs/1 Configuration/onSelectionChanging.md]:
```suggestion
A function that is called before selection changes.
```

File: api-reference/10 UI Components/dxTabs/1 Configuration/onSelectionChanging.md [hunk 14]
```diff
@@ -4,25 +4,25 @@ type: function(e)
...
-<!-- Description goes here -->
+The UI component's instance.
 
 ##### field(e.element): DxElement
-<!-- Description goes here -->
+#include common-ref-elementparam with { element: "UI component" }
 
 ##### field(e.removedItems): Array<any>
-<!-- Description goes here -->
+The data of the items whose selection has been canceled.
```
Comment (Abadzhev) [inline api-reference/10 UI Components/dxTabs/1 Configuration/onSelectionChanging.md]:
```suggestion
The data of the items removed from selection.
```

File: api-reference/10 UI Components/CollectionWidget/1 Configuration/onSelectionChanging.md [hunk 15]
```diff
@@ -5,28 +5,28 @@ default: null
...
-<!-- Description goes here -->
+A function that is called before selection changes.
 
 ##### param(e): Object
-<!-- Description goes here -->
+Information about the event.
 
 ##### field(e.addedItems): Array<any>
-<!-- Description goes here -->
+The data of the items that have been selected.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/CollectionWidget/1 Configuration/onSelectionChanging.md]:
```suggestion
The data of the selected items.
```

File: api-reference/10 UI Components/dxList/1 Configuration/onSelectionChanging.md [hunk 16]
```diff
@@ -4,25 +4,25 @@ type: function(e)
...
-<!-- Description goes here -->
+A function that is called before selection changes.
 
 ##### param(e): ui/list:SelectionChangingEvent
-<!-- Description goes here -->
+Information about the event.
 
 ##### field(e.addedItems): Array<any>
-<!-- Description goes here -->
+The data of the items that have been selected.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxList/1 Configuration/onSelectionChanging.md]:
```suggestion
The data of the selected items.
```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**